.upstream-tests/test/std/utilities/time/time.point/time.point.arithmetic/op_+=.pass.cpp

```diff
@@ -0,0 +1,47 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <chrono>
+
+// time_point
+
+// time_point& operator+=(const duration& d);
+// constexpr in c++17
+
+#include <cuda/std/chrono>
+#include <cuda/std/cassert>
+
+#include "test_macros.h"
+
+#if TEST_STD_VER > 14
```

Comment:
I assume this is consciously C++17 only? Or you just missed it? (Same in the next file.)

Reply:
Nope, looks like I missed it. I wasn't as diligent about checking test files for pre-processor C++ version checks.

Reply:
So I looked into this further. These tests are guarded with `TEST_STD_VER > 14` because they are testing functionality that wasn't made `constexpr` until C++17, e.g., https://en.cppreference.com/w/cpp/chrono/time_point/operator_arith

It doesn't appear there's any language reason these operations can't be made `constexpr` in C++14. Are we comfortable with backporting the `constexpr`ness of these operations from C++17 to C++14?

Reply:
Do you need them be constexpr in C++14? I _think_ it'd be fine if there's nothing C++17 specific in that code, given we are already backporting parts of this, but I'm unsure here.

Reply:
I don't think we have a strong need for this to be `constexpr`, but if we're already backporting, might as well?